default/python/AI/PolicyAI.py

```diff
@@ -189,44 +193,48 @@ def _can_adopt(self, name, replace_other=None):
             ret and self._empire.emptyPolicySlots[policy.category] and self._empire.policyPrereqsAndExclusionsOK(name)
         )
 
-    def _get_adoptable(self):
+    def _get_adoptable(self) -> set[str]:
```

Comment:
@Grummel7 
Compiled windows version has its own Python onboard, and Linux uses the system Python.  

This syntax is not supported by Python 3.6. I am fixing it for Builds.

Reply:
It's very strange, why CI has not caught this. 

Here is fix.
https://github.com/freeorion/freeorion/pull/3761